modules/templates/helper.go

```diff
@@ -267,6 +267,10 @@ func ToUTF8WithErr(content []byte) (string, error) {
 	if err != nil {
 		return "", err
 	} else if charsetLabel == "UTF-8" {
+		if len(content) > 2 && bytes.Equal(content[0:3], base.UTF8BOM) {
```

Comment:
How about to create a function named `RemoveUTF8BOM(content string) string`.

Reply:
Done